package.json

```
@@ -0,0 +1,30 @@
+{
+  "name": "we_count",
+  "version": "1.0.0",
+  "description": "We Count Project",
+  "author": "Ted Thompson",
+  "private": true,
+  "scripts": {
+    "dev": "nuxt",
+    "build": "nuxt build",
+    "start": "nuxt start",
+    "generate": "nuxt generate",
+    "lint": "eslint --ext .js,.vue --ignore-path .gitignore ."
```

Might be good to add a `lint:fix` command too:

```suggestion
    "lint": "eslint --ext .js,.vue --ignore-path .gitignore .",
    "lint": "eslint --ext .js,.vue --ignore-path .gitignore . --fix"
```